packages/cascara/src/system-components/Boundaries/fallbacks/ErrorFallback.js

```diff
@@ -8,18 +9,44 @@ const propTypes = {
 };
 
 const ErrorFallback = ({ error, resetErrorBoundary }) => {
+  const checkedFromLS =
+    window.localStorage.getItem('ESPRESSIVE_SHOW_ERRORS') === 'true';
+  const [checked, setChecked] = useState(checkedFromLS);
+  const showError = process.env.NODE_ENV === 'development' || checked;
+
+  const hangleCheckboxClick = useCallback(() => {
+    setChecked(!checked);
+  }, [checked]);
+
+  const hangleTryAgainClick = useCallback(() => {
+    window.localStorage.setItem('ESPRESSIVE_SHOW_ERRORS', checked);
+
+    if (resetErrorBoundary) {
+      resetErrorBoundary();
+    }
+  }, [checked, resetErrorBoundary]);
+
   return (
-    <div className='ui tiny error message' role='alert'>
+    <Role className='ui tiny error message' role='alert'>
       <div className='header'>Something went wrong:</div>
-      <pre>{error?.message}</pre>
+      {showError && <pre>{error?.message}</pre>}
+      <br />
```

Comment:
```suggestion
      {showError && <p><pre>{error?.message}</pre></p>}
```

Reply:
I did not commit the suggestion because I had more changes in my local, but this change has been applied. 👍